test/utilities.py

```diff
@@ -93,12 +97,14 @@ def thread_fn(target, args, result):
         for item in thread_return:
             yield item
 
-def check_DD_table_results(test, results, expected):
+
+def check_dd_table_results(test, results, expected):
     for result in results:
         for denomination in ['C', 'D', 'H', 'S', 'N']:
             for declarer in ['N', 'S', 'E', 'W']:
                 test.assertEqual(expected[denomination][declarer],
-                        result[denomination][declarer],
-                        declarer + ' should make ' + \
-                                str(expected[denomination][declarer]) + ' in ' + \
-                                denomination);
+                                 result[denomination][declarer],
+                                 declarer + ' should make ' +
+                                 str(expected[denomination][
+                                         declarer]) + ' in ' +
+                                 denomination)
```

Comment:
I see. I need a PEP-0008 compliant indent script for python.

But if ident rule goes like that then I would separate the assert call to a separate function which takes expected, result and declarer as argument. Then the argument indention would look better. I probably should have done that already in my commit. because indentation depth is too deep from the loops.

Reply:
You could write an iterator. I use them in my Javascript for the frontend. I don't think the indentation here is bad, though - it's easy to see what's going on.

Method extracted.